InventoryWatch/Model/Model.swift

```diff
@@ -461,15 +357,14 @@ final class Model: ObservableObject {
             guard let partsAvailability = storeJSON["partsAvailability"] as? [String: [String: Any]] else { return nil }
             let parsedParts: [PartAvailability] = partsAvailability.values.compactMap { part in
                 guard let partNumber = part["partNumber"] as? String else { return nil }
-                guard let storePickupProductTitle = part["storePickupProductTitle"] as? String else { return nil }
```

Comment:
this is the primary change in this PR, in addition to removing `storePickupProductTitle` from the `PartAvailability` struct.